frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.tsx

```diff
@@ -27,7 +28,17 @@ const EXPRESSIONS_DOCUMENTATION_URL = MetabaseSettings.docsUrl(
   "questions/query-builder/expressions",
 );
 
-export interface ExpressionWidgetProps {
+interface LegacyQueryProps {
```

Comment:
We could move `expression` and `onChangeExpression` props into this interface

Reply:
you mean those props make sense only with legacyQuery?